### PR TITLE
 add FLUSH option to boost log to control auto_flush

### PR DESCRIPTION
**Pull request by cyjseagull**
1. FLUSH=true: auto_flush
2. FLUSH=false: disable auto_flush

### CODE REVIEW

File: tools/build_chain.sh [hunk 1]
```diff
@@ -457,6 +457,8 @@ generate_config_ini()
     ;log level INFO DEBUG TRACE
     Level=${log_level}
     MaxLogFileSize=209715200
+    ; control log auto_flush
+    FLUSH=true
```
Comment (wheatli) [inline tools/build_chain.sh]: Flush

File: libinitializer/BoostLogInitializer.cpp [hunk 2]
```diff
@@ -61,8 +61,13 @@ void LogInitializer::initLog(
             boost::log::keywords::channel = channel));
     boost::shared_ptr<sink_t> sink(new sink_t(backend));
     /// set rotation size
-    uint64_t rotation_size = pt.get<uint64_t>(logType + ".MaxLogFileSize", 209715200);
+    uint64_t rotation_size = pt.get<uint64_t>("log.MaxLogFileSize", 209715200);
     sink->locked_backend()->set_rotation_size(rotation_size);
+
+    /// set auto-flush according to log configuration
+    bool need_flush = pt.get<bool>("log.FLUSH", true);
```
Comment (wheatli) [inline libinitializer/BoostLogInitializer.cpp]: Flush

File: tools/build_chain.sh (hunk 1 above)
Comment (cyjseagull) [reply tools/build_chain.sh]: All configurations related to log has been modified to low case.

File: libinitializer/BoostLogInitializer.cpp (hunk 2 above)
Comment (cyjseagull) [reply libinitializer/BoostLogInitializer.cpp]: All configurations related to log has been modified to low case.

**Pull request merged**